client/src/templates/Challenges/components/CompletionModalBody.js

```diff
@@ -58,7 +61,11 @@ export class CompletionModalBody extends PureComponent {
   }
 
   render() {
-    const { blockName, completedPercent, t } = this.props;
+    const { blockName, completedPercent, superBlock, t } = this.props;
+
+    const blockTitle = t(
+      `intro:${superBlock}.blocks.${dasherize(blockName)}.title`
```

Comment:
Rather than dasherizing (which I'm trying to remove entirely from the client), can we just pass `block` here?

Reply:
Should be fixed